solidity/contracts/test/MaintainerProxyStub.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity ^0.8.9;
+
+import "../maintainer/MaintainerProxy.sol";
+
+/// @title Maintainer Proxy Stub
+contract MaintainerProxyStub is MaintainerProxy {
+    constructor(Bridge _bridge, ReimbursementPool _reimbursementPool)
+        MaintainerProxy(_bridge, _reimbursementPool)
+    {}
+
+    function getAllMaintainers() external view returns (address[] memory) {
+        return maintainers;
```

Comment:
`maintainerProxy.maintainers()` won't get you an entire array. We can use `maintainerProxy.maintainers()` only if you pass a value to it, like `maintainerProxy.maintainers(0)`, otherwise it won't even compile. To get an entire array we need to expose a function just like `getAllMaintainers`

Reply:
So let's move `getAllMaintainers` to `MaintainerProxy`. This could be helpful to determine who is *before* and *after* in the queue.

Reply:
[d2a97c6](https://github.com/keep-network/tbtc-v2/pull/209/commits/d2a97c69cdebc19ed5b6aaa489982ea746bcbf5b)